app/views/hyrax/uploads/_js_templates.html.erb

```diff
@@ -0,0 +1,133 @@
+<!-- The template to display files available for upload -->
+<% fade_class_if_not_test = Rails.env.test? ? '' : 'fade' %>
+<script id="template-upload" type="text/x-tmpl">
+{% for (var i=0, file; file=o.files[i]; i++) { %}
+    <tr class="template-upload <%= fade_class_if_not_test %> in">
+        <td>
```

Comment:
I think line 5 is where the magic is happening, and I'm wondering if this could be a little opaque to our future selves?  Would it be worth adding a comment about how and why this is different from the corresponding file in Hyrax?